### PR TITLE
Add Bracknell Forest, UK & Braintree, UK.

### PR DESCRIPTION
**Pull request by cpressland**
Added source and documentation for Bracknell Forest, UK and Braintree, UK. Uses Post Code and House Number.

### CODE REVIEW

File: custom_components/waste_collection_schedule/waste_collection_schedule/source/bracknell_forest_gov_uk.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+            data={
+                "code_action": "find_rounds",
+                "code_params": json.dumps({"addressId": id}),
+            }
+            | self.data,
+        )
+        collection_lookup.raise_for_status()
+        collections = collection_lookup.json()["response"]["collections"]
+        entries = []
+        for waste_type in ["Food", "Recycling", "Garden", "General Waste"]:
```
Comment (mampfes) [inline custom_components/waste_collection_schedule/waste_collection_schedule/source/bracknell_forest_gov_uk.py:71]: This is the same list of waste types as in ICON_MAP. Would it make sense to merge them together?

**Review by mampfes — CHANGES_REQUESTED** [review]
Please add source also to info.md

**Comment by cpressland** [comment]
@mampfes - requested changes applied, also modified PR to include another source: Braintree, UK. Lemme know if you're happy with these or if you want it all in a single commit.

**Comment by mampfes** [comment]
Thanks for your contribution.

**Comment by mampfes** [comment]
> Lemme know if you're happy with these or if you want it all in a single commit.

Ok this time, but please create one PR per source next time.

**Pull request merged**